subjects/devops/play-with-containers-py/audit/README.md

```diff
@@ -0,0 +1,225 @@
+#### General
+
+##### Check the Repo content
+
+A `README.md` file and all files used to create, delete and manage the student infrastructure must be submitted in the repo.
+
+###### Are all the required files present?
+
+###### Does the `.env` file not exist in the repo?
```

Comment:
maybe this is better:
```"Is the .env file absent within the repository?```

Reply:
I opted for a slightly different version of yours :)